napari/_app_model/_app.py

```diff
@@ -38,8 +38,8 @@ def __init__(self, app_name=APP_NAME) -> None:
         self.menus.append_menu_items(SUBMENUS)
 
     @classmethod
-    def get_app(cls) -> NapariApplication:
-        return Application.get_app(APP_NAME) or cls()
+    def get_app(cls) -> NapariApplication:  # type: ignore[override]
+        return Application.get_app(APP_NAME) or cls()  # type: ignore[return-value]
```

Comment:
```suggestion
    def get_app(cls, app_name: str = APP_NAME) -> NapariApplication: 
        return Application.get_app(app_name) or cls()  # type: ignore[return-value]
```
I think this would be a small fix matching the signature of the class_method get_app in Application and thereby omitting the need for the ignore